app/model/profile.model.js

```diff
@@ -279,16 +269,10 @@ function loadCertificateSubEntities(headers) {
     return function(certificates) {
         return new Promise(function(resolve) {
             var skills = skillResource.getAllSkills(headers);
-            var customers = customerResource.getAllCustomers(headers);
-            var domains = domainResource.getAllDomains(headers);
-            return Promise.all([skills, customers, domains])
+            return Promise.all([skills])
```

Comment:
Promise.all is used when there are several promises to fulfill, which is not the case here. you can simply do:: 
return skillResource.getAllSkills(headers)
.then(function(skills) {
return loadSkillsForCertificates(headers, skills)(certificates);
}); 

no need for a wrapper promise